### ISSUE TITLE
Document db optimization using select_releated/prefetch_releated

### ISSUE THREAD

**Issue by HoshiYamazaki** [jazzband/sorl-thumbnail#668]
I'm curious if we are able to use `select_releated` or `prefetch_releated` techniques to optimize the load of the speed of view that uses sorl thumbnails templatetags.

**Comment by sowinski** [comment]
Did you find a way?

**Comment by HoshiYamazaki** [comment]
Nope, but because sorl thumbnail queries are not really taking so long, I don't care. 
I just optimize other parts of code for my webapps :)
